src/components/Input/Input.module.css

```diff
@@ -4,29 +4,17 @@
   border: var(--rvr-border-width-md) solid var(--rvr-gray-60);
   color: var(--rvr-gray-90);
   font-family: var(--rvr-base-font-family);
-  font-size: var(--rvr-font-size-md);
   line-height: var(--rvr-line-height-lg);
+  outline: none;
   padding: var(--rvr-space-bordered-sm);
-  transition:
-    100ms linear background-color,
-    100ms linear border-color,
-    100ms linear box-shadow,
-    100ms linear color;
+  transition: 100ms linear background-color, 100ms linear border-color,
```

Comment:
Stylelint is responsible for this. I believe this was something that should have been caught before, but was not being caught/fixed properly with the `.stylelintrc` file.